docs/_layouts/default.html

```diff
@@ -55,6 +55,17 @@
               <li><a href="{{ '/debugging.html' | relative_url }}">Debugging common Bolt errors</a></li>
             </ul>
           </li>
+          <li>
+            Bolt Developer Updates
+            <ul>
+              {% for post in site.posts %}
+                <li><a href="{{ post.url | remove: "/index.html" | relative_url }}">{{ post.date | date_to_string: "ordinal", "US" }} - {{ post.title }}</a></li>
```

Comment:
```suggestion
                <li><a href="{{ post.url | remove: "/index.html" | relative_url }}">{{ post.title }}</a><br><small>{{ post.date | date: "%m-%d-%y" }}</small></li>
```

This looks a little messy in the sidebar, especially when the title is long and wraps to the next line. If we want to display the date in the link I wonder if it makes sense to have it on a separate line?

Another approach is to add some CSS styling to these list items that adds ellipsis when the title reaches the width of the sidebar.

Reply:
I'm happy with this styling improvement. We can probably find something better, but I'll leave that for someone with greater CSS chops than I have.